src/libostree/ostree-sysroot-deploy.c

```diff
@@ -614,6 +615,83 @@ checkout_deployment_tree (OstreeSysroot *sysroot, OstreeRepo *repo, OstreeDeploy
                                 cancellable, error))
     return FALSE;
 
+#ifdef HAVE_COMPOSEFS
+  if (repo->composefs_wanted != OT_TRISTATE_NO)
+    {
+      static const char integrity_section[] = "ex-integrity";
```

Comment:
This one is probably better as
`#define _OSTREE_INTEGRITY_SECTION "ex-integrity"` in ostree-repo-private.h